src/Calculator/Views/GraphingCalculator/KeyGraphFeaturesPanel.xaml

```diff
@@ -308,7 +315,7 @@
         <converters:BooleanToVisibilityNegationConverter x:Name="BooleanToVisibilityNegationConverter"/>
     </UserControl.Resources>
 
-    <Grid Margin="0,7,0,0" Background="Transparent">
```

Comment:
Related to margins but not to this PR, the KGF ListView is 10 pixels from the right side of the window, can you update its margin in this PR? 

```
   <ListView x:Name="KeyGraphFeaturesListView"
                  Grid.Row="2"
                  Margin="12,10,10,0"
                  Padding="0,0,0,12"
```
 should become
```
   <ListView x:Name="KeyGraphFeaturesListView"
                  Grid.Row="2"
                  Padding="12,10,10,12"
```

Reply:
![image](https://user-images.githubusercontent.com/1226538/70696477-a352fe80-1c78-11ea-879d-c3277b05e245.png)

Reply:
Good eye!